packages/block-editor/src/components/list-view/use-list-view-scroll-into-view.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { getScrollContainer } from '@wordpress/dom';
+import { useLayoutEffect } from '@wordpress/element';
+
+export default function useListViewScrollIntoView( {
+	isSelected,
+	selectedClientIds,
+	rowItemRef,
+} ) {
+	const numBlocksSelected = selectedClientIds?.length;
+
+	useLayoutEffect( () => {
+		// Skip scrolling into view if more than one block is selected.
+		// This is to avoid scrolling the view in a multi selection where the user
+		// has intentionally selected multiple blocks within the list view,
+		// but the initially selected block may be out of view.
+		if (
+			! isSelected ||
+			numBlocksSelected !== 1 ||
+			! rowItemRef?.current ||
+			! rowItemRef.current.scrollIntoView
```

Comment:
Not sure why this is needed? `scrollIntoView` should be available for all browsers that we support.

Reply:
Great point, I'd seen a check for it elsewhere in GB but that's probably from a long time ago and doesn't appear to be needed now 👍